lib/resultsCapturer.js

```diff
@@ -58,15 +58,13 @@ module.exports = function () {
   });
 
   this.registerHandler('StepResult', function (event, callback) {
```

Comment:
I would suggest renaming the variable to what it actually is aka from event to stepResult in the function arguments. The same applies for the other handlers.